department-of-veterans-affairs/caseflow
# Queue | Save issue edits to server

Closes #5035

### Description
After editing an issue, this PUTs the changes to the server

### Acceptance Criteria 
- [ ] Edited issue information persists between sessions/page loads

### Testing Plan
1. Go to `/queue` -> task -> Decision Ready for Review -> Edit Issue
2. Make changes to the issue, Save, verify you're returned to Select Issue Dispositions page
3. Reload `/queue/${taskId}`, confirm issue information is updated
